app/services/api/v1/ecf_participants/index.rb

```diff
@@ -74,47 +69,6 @@ def updated_since
         rescue ArgumentError
           Time.iso8601(URI.decode_www_form_component(filter[:updated_since]))
         end
-
-        # inner most query
-        # this query deals with the following scenario
-        # given one profile with 2 induction records
-        # we only want to return the latest induction record
-        # we also exclude any partnerships that have been challenged
-        def induction_record_ids_with_deduped_induction_records
```

Comment:
we have lost this de-dupe. therefore it should be noted that if a user has more than one profile:
- before this change there would be one record returned for a user
- after this change we now show as many records as they have profiles